WordPress/Classes/ViewRelated/Post/PostEditor.swift

```diff
@@ -144,3 +148,9 @@ extension PostEditor {
         return [.visibility, .schedule, .tags, .categories]
     }
 }
+
+enum PostEditorEntryPoint: String {
+    case unknown
+    case list
```

Comment:
What do you think of using `siteMenu` instead of `list`? I'm worried that list sounds too generic

Reply:
`siteMenu` makes me think of the FAB or the site menu under "My Site". 😬

`list` means the Posts List screen, maybe I should change to `postsList`? Thoughts?

Reply:
Oh! Yes, `postsList` sounds great.

Reply:
Chnged in d6950a974eeea103e375ea91cf2ab5168fb71f6f